tests/foreman/api/test_errata.py

```diff
@@ -6,7 +6,7 @@
 
 :CaseLevel: Acceptance
```

Comment:
Since all tests in this file overwrite it to set `:CaseLevel: System`, you might as well change this to `System` here and remove assignments on individual test level.
But you don't have to, effect is the same.